package/yast2-bootloader.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Fri May  4 12:43:44 UTC 2018 - jreidinger@suse.com
+
+- Inform user when bootloader is in supported in autoyast profile
```

Comment:
"Inform user when an unsupported bootloader is defined in the AutoYaST profile, display the valid values"